src/main/java/com/nike/cerberus/operation/cms/CreateCmsCmkOperation.java

```diff
@@ -18,6 +18,7 @@
 
 import com.amazonaws.regions.Regions;
 import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.internal.Maps;
```

Comment:
I think you probably meant to use Guava and not an internal thing of Jcommander

Reply:
DOH!